packages/react/src/components/AccountSettings/SetupTOTP/defaults.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Alert, Button, Image, TextField } from '../../../primitives';
+import {
+  ButtonComponent,
+  ErrorMessageComponent,
+  ImageComponent,
+  SubmitButtonComponent,
+  TextFieldComponent,
+} from '../types';
+
+export const ConfirmationCode: TextFieldComponent = ({ label, ...rest }) => (
+  <TextField {...rest} label={label} />
+);
```

Comment:
This is one weird part that I need TS advice on. Instead of directly assigning this like

```ts
ConfirmationCode: TextFieldComponent = TextField`
```

I had to separate out `label` to go around the error `label` is required but is optional in `TextFieldComponent`. I wanted to keep `label` optional because it's a UI prop and IMO shouldn't be required for override components.

Reply:
I tried experimenting with `Omit` and adding default params, but I couldn't get it to work. I think the way you did it is fine.

Reply:
That's fair. 
- making label required: 6c1dfc08b955e979e9cbf56e6ba7fd24a4b74d4c
- renaming this field: 68cafa239d9cc4f64f18e26aae41d05e53268463